pages/index.tsx

```diff
@@ -23,7 +23,7 @@ const Home: NextPage = () => {
     <>
       <Head>
         <title>
-          Maryland Tree Trimming & Removal Services | Kuhns Tree Service{" "}
+          Tree Trimming & Removal Services in Silver Springs, MD | Kuhns Tree Service{" "}
```

Comment:
Total typo. I spelled Silver Spring wrong.

Reply:
I think it's supposed to be `Silver Spring`. We've been having the same problem remembering 😄

Reply:
```suggestion
          Tree Trimming & Removal Services in Silver Spring, MD | Kuhns Tree Service{" "}
```